backend/src/test/java/io/tribehouse/motomo/MotomoApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package io.tribehouse.motomo;
```

Comment:
why we have another copy of the same class?

Reply:
Removed unused class.